GitUI/CommandsDialogs/FormBrowseController.cs

```diff
@@ -26,6 +29,26 @@ public class FormBrowseController : IFormBrowseController
         private readonly IRepositoryCurrentBranchNameProvider _repositoryCurrentBranchNameProvider;
         private readonly IInvalidRepositoryRemover _invalidRepositoryRemover;
 
+        private static readonly Dictionary<RepoType, Bitmap> _repoTypeBitmaps = new Dictionary<RepoType, Bitmap>()
+        {
+            { RepoType.Current, Properties.Images.FolderOpen },
```

Comment:
Yet another status-> icon resolver... Slightly different from the more similar submodule icon resolvers though, I guess it has to be separate

Reply:
Could you please point such an icon resolver?

Reply:
Search for SubmoduleRevisionSemiDownDirty and you find resolvers in FormBrowse, RepoObjectTree, FileStatusList

Reply:
But it isn't separated in `FormBrowse` for instance - just single [`GetSubmoduleItemImage`](https://github.com/gitextensions/gitextensions/blob/5dd2d773a942a586189f240bb69f89bb64eb4064/GitUI/CommandsDialogs/FormBrowse.cs#L2723) function is used.